src/docs/common/snippets/common-ksuid.adoc

```diff
@@ -0,0 +1,8 @@
+=== Ksuid
+
+Add the following dependency to generate KSUIDs (K-Sortable Globally Unique IDs).
+____
+KSUID is for K-Sortable Unique IDentifier. It's a way to generate globally unique IDs similar to RFC 4122 UUIDs, but contain a time component so they can be "roughly" sorted by time of creation. The remainder of the KSUID is randomly generated bytes.
```

Comment:
KSUID is for K-Sortable Unique IDentifier. It's a way to generate globally unique IDs similar to RFC 4122 UUIDs that contain a time component, so they can be "roughly" sorted by time of creation. The remainder of the KSUID is made up of randomly generated bytes.